openmc/material.py

```diff
@@ -1316,6 +1322,40 @@ def get_mass(self, nuclide: str | None = None, volume: float | None = None) -> f
             raise ValueError("Volume must be set in order to determine mass.")
         return volume*self.get_mass_density(nuclide)
 
+    def waste_classification(self, metal: bool = False, method: str = 'NRC') -> str:
```

Comment:
I'd make the first arg here `method` rather than `metal` and then potentially `**kwargs` which gets passed to each method. This would allow you to pass in an alternative dictionary as @gonuke suggests and is generally extensible for other methods. Plus the `metal` kwarg isn't relevant to the Fetter classification scheme.

Reply:
Good suggestion. I decided not to go with the `**kwargs` approach because it makes things a little less obvious to a user who is calling `Material.waste_classification` (would force them to look at those internal functions to figure out what arguments are available).